src/blockly/generators/propc.js

```diff
@@ -222,7 +222,7 @@ function setProfile(profileName) {
     }
 
     // Setting a default baud rate
```

Comment:
This is the only place where the setBaudrate function was used.  All of the other uses access the global variable directly.  While this a a small step backward, I think the baudrate can be held by the board profile object, since that's where it is sourced anyway.

Reply:
Setting a global baud rate doesn't make sense to me. There should be a baud rate setting for each device that supports serial I/O and that probably belongs to the device, and not the board type. The board type can host a default setting. This assumes that there can be multiple devices in a project that support serial I/O.

Reply:
It's not necessarily global - this is just the baudrate setting for the terminal, it just happens to be used across multiple files, so it's a global variable until we can move it to a class.  An issue for telling the comms to use the device baudrate property on open would be a good idea.